tests/metaworld/envs/mujoco/sawyer_xyz/base.py

```diff
@@ -0,0 +1,61 @@
+from metaworld.policies.base import PID
+
+
+def scripted_policy_test(env, policy, label, iters=1000):
+    """Tests whether a given policy solves an environment in a stateless manner
+
+    Args:
+        env: Environment to test
+        policy: Policy that's supposed to succeed in env
+        label: A string to use when printing out errors
+        iters: How many times the policy should be tested
+
+    """
+    for name, value in vars(policy).items():
+        assert isinstance(value, PID), label + ' policy has potential state variable: ' + name
+        assert value.stateful is False, label + ' policy has PID with stateful=True: ' + name
+
+    for i in range(iters):
+        ensure_success(env, policy)
+
+
+def ensure_success(env, policy, render=False, retries=1):
+    """Tests whether a given policy solves an environment
+
+    Args:
```

Comment:
this is godo documentation. if you can please also specify the types for args (e.g. `env (gym.Env): Environment to test.`)